.github/workflows/build.yml

```diff
@@ -0,0 +1,220 @@
+        with:
+          draft: true
+          files: |
+            build/libs/*.jar
+          name: ${{ steps.prepare.outputs.tag_name }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  sonar:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        java:
+          - 11
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: Dump GitHub context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: echo "$GITHUB_CONTEXT"
+      -
+        name: Dump job context
+        env:
+          JOB_CONTEXT: ${{ toJson(job) }}
+        run: echo "$JOB_CONTEXT"
+      -
+        name: Dump steps context
+        env:
+          STEPS_CONTEXT: ${{ toJson(steps) }}
+        run: echo "$STEPS_CONTEXT"
+      -
+        name: Dump runner context
+        env:
+          RUNNER_CONTEXT: ${{ toJson(runner) }}
+        run: echo "$RUNNER_CONTEXT"
+      -
+        name: Set up Java ${{ matrix.java }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+          java-package: jdk
+      -
+        name: Cache deps
+        uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      -
+        name: Sonar
+        run: |
+          if [[ $GITHUB_REF == refs/tags/* ]]; then
+            ./gradlew sonarqube \
+              -Dsonar.pullrequest.provider=github \
+              -Dsonar.pullrequest.github.repository=${{ github.repository }} \
+              -Dsonar.host.url=https://sonarcloud.io \
+              -Dsonar.organization=${SONAR_ORG_KEY} \
+              -Dsonar.projectKey=${SONAR_PROJECT_KEY} \
+              -Dsonar.projectVersion=${GITHUB_REF#refs/tags/} \
+              -Dsonar.branch.name=master
+          elif [ "${{ github.event_name }}" = "pull_request" ]; then
+            PR_BRANCH=${{ github.head_ref }}
+            ./gradlew sonarqube \
+              -Dsonar.pullrequest.provider=github \
+              -Dsonar.pullrequest.github.repository=${{ github.repository }} \
+              -Dsonar.host.url=https://sonarcloud.io \
+              -Dsonar.organization=${SONAR_ORG_KEY} \
+              -Dsonar.projectKey=${SONAR_PROJECT_KEY} \
+              -Dsonar.pullrequest.key=${{ github.event.number }}
+              -Dsonar.pullrequest.base=${{ github.event.base_ref }}
+          else
+            ./gradlew sonarqube \
+              -Dsonar.pullrequest.provider=github \
+              -Dsonar.pullrequest.github.repository=${{ github.repository }} \
+              -Dsonar.host.url=https://sonarcloud.io \
+              -Dsonar.organization=${SONAR_ORG_KEY} \
+              -Dsonar.projectKey=${SONAR_PROJECT_KEY} \
+              -Dsonar.branch.name=${GITHUB_REF##*/}
+          fi
```

Comment:
@mc1arke I have made some changes to handle Sonarcloud parameters correctly. Now we handle all branches, pull request and releases: https://sonarcloud.io/dashboard?id=crazy-max_sonarqube-community-branch-plugin

![image](https://user-images.githubusercontent.com/1951866/83958367-571bfc00-a860-11ea-9382-644cdc6d982e.png)

![image](https://user-images.githubusercontent.com/1951866/83958370-5d11dd00-a860-11ea-873d-499ef2fa4797.png)

![image](https://user-images.githubusercontent.com/1951866/83958373-61d69100-a860-11ea-8f3b-cfae50f5a582.png)